tests/platform_helper/domain/test_codebase.py

```diff
@@ -397,19 +385,6 @@ def test_codebase_deploy_does_not_trigger_build_with_missing_environment(mock_ap
         )
 
 
-def test_codebase_deploy_does_not_trigger_deployment_without_confirmation():
```

Comment:
Are we no longer doing confirmation?

Reply:
This looked like a duplicate test. There is a test for "without_confirmation" [here](https://github.com/uktrade/platform-tools/blob/9874dc7239872a6f79efeee0307cfa28ff1057b6/tests/platform_helper/domain/test_codebase.py#L338)